scripts/create-component.js

```diff
@@ -2,32 +2,42 @@
 const mustache = require('mustache');
 const argv = require('yargs').argv;
 const newComponentName = argv.name;
+const stateless = argv.stateless;
 const fs = require('fs');
 
+// Template Sequence
+const statefulSequence = ['GlobalIndex', 'Styles', 'Index', 'View', 'ViewTest', 'Types', 'Component', 'State', 'StateTest'];
+const statelessSequence = ['GlobalIndex', 'Styles', 'Index', 'View', 'ViewTest', 'TypesStateless', 'ComponentStateless'];
```

Comment:
Nit, but it might be worth sharing a common sequence base that you concat to. I'm unsure how much you anticipate these sequences changing so feel free to disregard.